docs/connecting.txt

```diff
@@ -114,4 +115,95 @@ CrateDB JDBC driver supports following properties:
     only supported transaction isolation level is ``TRANSACTION_NONE``.
     Read-only connections aren't supported either.
 
+:user: ``<String>``
+
+    The database user on whose behalf the connection is being made.
+
+:password: ``<String>``
+
+    The database user's password.
+
+:ssl: ``true``
+
+    Connect using SSL. The driver must have been compiled with SSL support.
```

Comment:
Would remove the second sentence. AFAIK we make sure the driver includes SSL support.

Reply:
Good catch!